spec/shoryuken/worker_spec.rb

```diff
@@ -66,6 +66,19 @@
       TestWorker.perform_async('message')
     end
 
+    it 'enqueues a message given as hash' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.